### ISSUE TITLE
newl doesn't work even though it is in the readme

### ISSUE THREAD

**Issue by ghost** [Antosser/brainfuck-compiler-web#6]
```
(Line: 7) Error: Command newl does not exist.
```

I also noticed that in the "commands" dictionary in the code it does not contain the newl command, which may be something to do with it..

**Comment by ghost** [comment]
Note: a "printl" command with no argument works as a replacement. Example:

```
var coolvalue
set coolvalue 69
var badvalue
set badvalue 21
switch badvalue coolvalue
printdec badvalue
printl
printdec coolvalue
```

Outputs:
```
69
21
```

**Comment by Antosser** [comment]
It was a really long time ago, but I do remember `newl` being a thing. I probably removed it somewhere along the way. Thanks for the interest and noticing!

**Comment by Antosser** [comment]
Fixed in [43b50fa](https://github.com/Antosser/brainfuck-compiler-web/commit/43b50fa084b53786397b502f642b3b999175b463). Thx!

**Comment by ghost** [comment]
> Fixed in [43b50fa](https://github.com/Antosser/brainfuck-compiler-web/commit/43b50fa084b53786397b502f642b3b999175b463). Thx!

Awesome!
